src/reducers/index.js

```diff
@@ -2,10 +2,12 @@ import { combineReducers } from 'redux';
 
 import alertsReducer from '../components/AlertsNotifications/alerts-reducer';
 import loginReducer from '../components/LogIn/LoginReducer';
+import ShipDetailsMapMapReducer from '../components/GlobalMap/ShipDetailsMap/ShipDetailsMapReducer'
 
 
 export default combineReducers({
     alertsReducer,
-    loginReducer
+    loginReducer,
+    ShipDetailsMapMapReducer
```

Comment:
There is this song in Serbia that repeats the word map all the time (mapa, mapa, mapa...). It is possible that I got influenced by it, and now I started writing the word map twice :) Will fix it